src/CFamily/CMakeProjectTypeIndicator.cs

```diff
@@ -58,11 +57,6 @@ public bool IsCMake()
 
             var rootDirectory = folderWorkspaceService.FindRootDirectory();
 
-            if (string.IsNullOrEmpty(rootDirectory))
```

Comment:
This is actually a change of behaviour, but it probably doesn't make any difference in practice - an different exception will still be thrown when we try to use the null value.

Reply:
It will still be an ArgumentNullException, coming from the directory enumerate. I left the test that verifies it.